src/components/FavoriteItem/FavoriteItem.js

```diff
@@ -110,48 +106,52 @@ class FavoriteItem extends PureComponent<Props, ComponentState> {
           <div className='actions'>
             {!isWalletReadOnly && (
               <Link
-                onMouseEnter={this.onHover('send')}
-                onMouseLeave={this.onHover(null)}
                 to={`/digital-assets/send?to=${address}`}
                 className='item -send'
               >
                 <JTooltip text={t`Send`}>
-                  <JIcon
-                    color={(hovered === 'send') ? 'sky' : 'blue'}
-                    name='upload'
-                  />
+                  <JIcon name='upload' />
                 </JTooltip>
               </Link>
             )}
             <Link
-              onMouseEnter={this.onHover('edit')}
-              onMouseLeave={this.onHover(null)}
               to={`/favorites/address/${address}`}
               className='item -edit'
             >
               <JTooltip text={t`Edit`}>
-                <JIcon
-                  color={(hovered === 'edit') ? 'sky' : 'blue'}
-                  name='edit'
-                />
+                <JIcon name='edit' />
               </JTooltip>
             </Link>
-            <div
-              onClick={this.onHover(null)}
-              onMouseLeave={this.onHover(null)}
-              onMouseEnter={this.onHover('trash')}
-              className='item -delete'
-            >
-              <ButtonWithConfirm
-                onClick={ignoreEvent(this.remove)()}
-                iconTooltipColor={(hovered === 'trash') ? 'sky' : 'blue'}
-                color='blue'
-                bgColor='white'
-                labelCancel={t`No`}
-                iconTooltipName='trash'
-                labelConfirm={t`Yes, delete`}
-                isReverse
-              />
+            <div className='item -delete'>
+              <div className='confirms'>
+                {
+                  this.state.showDelete ?
```

Comment:
copy-paste the same code in several places instead of re-using ButtonWithConfirm component?

Reply:
It's not same scenarios. Because all cases (3) of using this component is different way, and, in my opinion, more straightforward design is: when we make three different implementation.